app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -106,6 +106,9 @@ class NewMessageViewModel @Inject constructor(
     val uiQuoteLiveData = MutableLiveData<String?>()
     //endregion
 
+    var subjectTextField = ""
+    var bodyTextField = ""
```

Comment:
Maybe we should even make these two variables names clearer. The way it is right now, someone might think at first that these will always contain the correct subject and body which is not the case

Reply:
lastOnStop...?